### ISSUE TITLE
CalendarView in SwiftUI create menu items

### ISSUE THREAD

**Issue by mnightingale** [rechsteiner/Parchment#498]
I've been trying to adapt the Calendar example for SwiftUI based on PageView.

I currently have https://gist.github.com/mnightingale/d5a214f8eae1ec4111d6b2008f5a69f9 (mines a little different because I need to pick by week not day)
So the callback is used to create the page content in PagingViewControllerInfiniteDataSource using a SwiftUI view wrapped in a UIHostingController.

But `pagingViewController.register(CalendarPagingCell.self, for: CalendarItem.self)` registers the menu cells, is there a way I can get that to use a SwiftUI view via another UIHostingController?

Basically I'd like to dynamically create the menu and page with SwiftUI views.

**Comment by rechsteiner** [comment]
Hi @mnightingale! There's not built-in way of using SwiftUI for the menu views, but it should be possible if you create your own `PagingCell` subclass that embeds a SwiftUI view
